core/test/Cardano/Address/Style/ShelleySpec.hs

```diff
@@ -166,53 +163,36 @@ spec = do
             ,  verKeyStake = "1c2c3c4c5c6c7c8c"
             ,  netTag = 0
             ,  expectedAddr =
-                    "008a4d111f71a79169c50bcbc27e1e20b6e13e87ff8f33edc3cab419d4\
-                    \08b2d658668c2e341ee5bda4477b63c5aca7ec7ae4e3d196163556a4"
+                    "008a4d111f71a79169c50bcbc27e1e20b6e13e87ff8f33edc3cab419d408b2d658668c2e341ee5bda4477b63c5aca7ec7ae4e3d196163556a4"
             }
         goldenTestBaseAddress GoldenTestBaseAddress
             {  verKeyPayment = "1a2a3a4a5a6a7a8a"
             ,  verKeyStake = "1c2c3c4c5c6c7c8c"
             ,  netTag = 1
             ,  expectedAddr =
-                    "018a4d111f71a79169c50bcbc27e1e20b6e13e87ff8f33edc3cab419d4\
-                    \08b2d658668c2e341ee5bda4477b63c5aca7ec7ae4e3d196163556a4"
+                    "018a4d111f71a79169c50bcbc27e1e20b6e13e87ff8f33edc3cab419d408b2d658668c2e341ee5bda4477b63c5aca7ec7ae4e3d196163556a4"
             }
 
     describe "Test vectors" $ do
         testVectors TestVector
             { rootXPrv =
-                    "root_xprv1nq49elkxhet9d03y0uqxp5fa802jgqauef3878ve3r9xvkg8\
-                    \c4qqnq98raswxtrwlfw7txa7m49d6x84092l0lsv5s603dkakj8vwgsz23\
-                    \drjsarjmxdev8mnrllmxykxh7exsrtqn64d6j5qdqp9deqpqx70t25"
+                    "root_xsk1nq49elkxhet9d03y0uqxp5fa802jgqauef3878ve3r9xvkg8c4qqnq98raswxtrwlfw7txa7m49d6x84092l0lsv5s603dkakj8vwgsz23drjsarjmxdev8mnrllmxykxh7exsrtqn64d6j5qdqp9deqpqlu52ew"
```

Comment:
These changes seems both unnecessary and wrong? 

- The code was formatted to fit nicely on 80 columns.  
- Nothing done in this PR should affect these tests. If the test are failing, there's likely something wrong with the new implementation and it's the implementation that should be fixed, not the tests :grimacing:

Reply:
I guess the change comes from the prefix changes from `root_xprv` to `root_xsk`, which changes the entire encoding (and similarly for acct and test like). 
In such case, it'd be _okay_ to adjust the tests indeed. However, unless strictly necessary, I'd avoid touching them as much as possible. 

This could be done in the test itself, by changing the prefix to what's expected by the test (with a little note explaining the rationale). This way, we do still test the new implementation, but keep tests the same (which ease our minds because it doesn't look like we've broken everything).

Reply:
> The code was formatted to fit nicely on 80 columns.

so I could easily `sed` with copy/paste. It was about the limit of my frustration tolerance. I could replace them with a golden test.

> prefix changes

Yeah, they change the prefixes, and hence the checksum changes.

> This could be done in the test itself, by changing the prefix to what's expected by the test (with a little note explaining the rationale).

That sounds like trouble down the line, so I'd prefer avoiding it. I'd go with a golden test here so you can compare the output from the old one with the new one side-by-side. Or I could write myself a little macro to reformat the current code, seems to be mechanical enough.